luftfilterbegehren/templates/act/conversation_guide.html

```diff
@@ -0,0 +1,79 @@
+{% extends "layout.html" %}
+{% set active_page = "general.conversation_guide" %}
+
+{% block title %}
+    <title>Gesprächsleitfaden | {{ config['PROJECT_NAME'] }}</title>
+{% endblock title %}
+
+{% block content %}
+    <section class="chat">
+        <div class="container">
+            <h1>HILFE! Meine Gemeinde hat geantwortet!</h1>
+            <h2>Gesprächsleitfaden</h2>
+            <p class="center">Keine Angst, dass Gemeinden mit ihren Bürger*innen ins Gespräch kommen, ist erstmal begrüßenswert. Hier haben wir für dich ein paar Tipps und Informationen zusammengetragen, wie du ruhig reagieren kannst, wenn dir deine Gemeinde geantwortet hat.<p>
```

Comment:
"für dich" -> "für Sie".
"wie du" -> "wie Sie" -- bitte im ganzen File prüfen/korrigieren, in den Mails verwenden wir auch "Sie".